bee-inx/CHANGELOG.md

```diff
@@ -21,6 +21,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## 1.0.0-beta.6 - 2022-XX-XX
 
+### Added
```

Comment:
Are you going to release this version right after? If yes, you can bump the version in this PR, it's fine.

Reply:
Nah, we'll need to update the other crates before we update yours anyway